sc-tools/sc-builder/tests/units/test_gwf_translator.cpp

```diff
@@ -8,30 +8,142 @@
 
 #include "builder_test.hpp"
 
+#include <filesystem>
+
 #include "sc-memory/sc_utils.hpp"
 #include "../../src/gwf_translator.hpp"
+#include "../../src/sc_scs_tree.hpp"
+
+#define BASE_TEST_PATH "/home/iromanchuk/sc-machine/sc-tools/sc-builder/tests/kb/tests-gwf-to-scs/"
```

Comment:
I guess you can use `SC_BUILDER_KB "/tests-gwf-to-scs/"` instead of complete path